docs/dashboards/README.md

```diff
@@ -0,0 +1,237 @@
+
+The dashboards rely on metrics from the collectors' [internal telemetry](https://opentelemetry.io/docs/collector/internal-telemetry/). See the [list of internal metrics](https://opentelemetry.io/docs/collector/internal-telemetry/#lists-of-internal-metrics) for an overview of which metrics are available.
+
+In the example configuration for the selfmonitoring collector below, all selfmonitoring metrics are prefixed with `sfm.otelcol`. The dashboards expect this prefix and won't show metrics that do not have that prefix.
+
+## Prerequisites
+The dashboards rely on the selfmonitoring capabilities of the OTel collector as well as certain attributes on the exported metrics data.
+Required attributes are: 
+- `service.name` (automatically added by the Collector)
+- `service.instance.id` (automatically added by the collector, needs to be added to the Dynatrace attribute allow list; see "[Adding `service.instance.id` to the allow list](#adding-serviceinstanceid-to-the-allow-list)")
+- `k8s.pod.name` (needs to be added to the telemetry data; see the [Kubernetes section](#kubernetes) below)
+
+### Adding `service.instance.id` to the allow list
+`service.name` and `k8s.pod.name` are on the Dynatrace OTLP metrics ingest allow list by default, `service.instance.id` is not. In order to add it, follow [this guide](https://docs.dynatrace.com/docs/shortlink/metrics-configuration#allow-list) and add `service.instance.id` to the list.
+This will ensure that this resource attribute is stored as a dimension on the metrics in Dynatrace. 
+
+### Dynatrace ingest
+In order to send data to Dynatrace via OTLP, you will need to supply a Dynatrace endpoint and an ingest token with the `metrics.ingest` scope set. See the [Dynatrace docs](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/otlp-export) for more information.
+
+## Architecture
+Every OpenTelemetry collector has selfmonitoring capabilities, but they need to be activated.
+Selfmonitoring data can be exported from the collector via the OTLP protocol.
+The suggested way of exporting selfmonitoring data is to run one collector dedicated to collecting and exporting the selfmonitoring data for the other running collectors, and forwarding that data to Dynatrace.
+See the diagram below for an example architecture.
+The `sidecar-collector` and `gateway-collectors` are used to send application telemetry to Dynatrace (e.g. traces, metrics, logs that the application produces).
+Both of these collectors send only their selfmonitoring data (or internal telemetry) to the `selfmon-collector`.
+The `selfmon-collector` is responsible for collecting, transforming and forwarding the internal telemetry for all collectors (including the `selfmon-collector` itself).
+Only monitoring data about the collector passes through it; no monitoring data from non-collector applicaitons is sent via the `selfmon-collector`.
+
+```mermaid
```

Comment:
This picture is difficult to read when having GitHub configured for the dark background.